src/poliastro/czml/extract_czml.py

```diff
@@ -29,8 +29,8 @@ def __init__(self, start_epoch, end_epoch, N):
             of sampled data points will be N when calling
             add_orbit()
         """
-        self.czml = dict()
-        self.orbits = []
+        self.czml = dict()  # type : dict
```

Comment:
```suggestion
        self.czml = dict()  # type: dict
```

Otherwise, mypy doesn't understand it :sweat_smile: